README.md

```diff
@@ -431,6 +431,58 @@ One can request that mutable table is in one of three states:
  * `CREATED` - table is created but is not populated with data
  * `LOADED` - table is created and populated with data
 
+#### LdapObjectRequirement
+
+When this requirement is fulfilled, it will create an LDAP Entry in the configured LDAP Server. Only
+`OpenLDAP` LDAP server is supported now.
+
+The tempto configuration yaml must have the following configuration for this requirement:
+
+```ldap:
+     url: ldap://ldapserverhost
+     admin:
+       dn: cn=admin,dc=tempto,dc=com
+       password: admin
+```
+
+The LDAP Server used must be configured with an admin user and password who has privileges to create entities.
+If a similar entry already exists in the LDAP server, then the fulfiller will ignore adding that entry.
```

Comment:
To what entry does this line relate? It seems to me that this not should be part of `LdapObjectDefinition` documentation.